src/lib/x509/x509_ext.h

```diff
@@ -500,6 +500,66 @@ class OCSP_NoCheck final : public Certificate_Extension {
       void decode_inner(const std::vector<uint8_t>&) override;
 };
 
+/**
+* TNAuthList extension
+*
+* RFC8226 Secure Telephone Identity Credentials
+*   https://www.rfc-editor.org/rfc/rfc8226#section-9
+*/
+class BOTAN_PUBLIC_API(3, 6) TNAuthList final : public Certificate_Extension {
+   public:
+      class TNEntry final : public ASN1_Object {
+         public:
+            enum Type { TN_ServiceProviderCode = 0, TN_TelephoneNumberRange, TN_TelephoneNumber };
```

Comment:
Given that this is burried so deep into a class structure, I think we can live without the `TN_` prefix here.

Reply:
I believe you missed this. :)